src/components/Layout/Footer/Footer.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { FC } from "react";
+
+import "@mozilla-protocol/core/protocol/css/protocol.css";
+
+interface Link {
+  /*Displayed string represents the link*/
+  label: string;
+  /*URL to which the link redirects*/
+  url: string;
+}
+
+interface FooterLinksSection {
+  /*Array of links to be displayed below the 
+  footerLinkSectionName*/
+  links: Link[];
+  /*Name to be displayed at the top of the footer
+  column*/
+  footerLinksSectionName: string;
+}
+
+export interface FooterProps {
+  /*Array of footer columns, each with a name to be displayed at 
+  the top of the column, and an array of links to be displayed 
+  below the name*/
+  footerLinksSections: FooterLinksSection[];
+}
+
+function renderFooterLinksSection(footerLinksSection: FooterLinksSection) {
+  return (
+    <section className="mzp-c-footer-section">
+      <h5 className="mzp-c-footer-heading">{footerLinksSection.footerLinksSectionName}</h5>
+      <ul>
+        {footerLinksSection.links.map((link) => {
+          return (
+            <li key={link.label}>
+              <a href={link.url}>{link.label}</a>
+            </li>
+          );
+        })}
+      </ul>
+    </section>
+  );
+}
+
+const Footer: FC<FooterProps> = ({ footerLinksSections }: FooterProps) => {
+  return (
+    <footer className="mzp-c-footer">
+      <div className="mzp-l-content">
+        <nav className="mzp-c-footer-primary">
+          <div className="mzp-c-footer-sections">
+            <section className="mzp-c-footer-section">
+              <h5 className="mzp-c-footer-heading">About</h5>
+              <ul>
+                <li>
+                  <a href="https://councildataproject.github.io/#about">
+                    CouncilDataProject Information
```

Comment:
Nit: Change this to just "CouncilDataProject" and remove the pointer to `#about`.

Reply:
Should have caught that. I'll make sure to look at the bottom of storybook more closely in the future. 

The accessibility issue I'm getting reads `The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable`. I fixed this by adding `aria-label="footer"` as an attribute on the `nav` element (storybook provided a [link](https://dequeuniversity.com/rules/axe/4.1/landmark-unique?application=axeAPI) with more details and potential solutions just above the accessibility issue; I took this solution from that link). 

I tried adding the attributes `id="footer" role="contentinfo"`, but storybook was throwing me many errors. I think the latest commit should get rid of the issue, but let me know if you're still getting an error on your end when you look at it. Newest commit is also posted to [my gh-pages](https://andrewlubrino.github.io/cdp-frontend/)

Reply:
Beautiful. Thanks for resolving. I will will wait for To or Hawk to review but all good from my side. Whenever someone else approved will happily merge. Thanks for all your work on this :slightly_smiling_face: